common/ethnicities/wc_ethnicities_orcish_rework.txt

```diff
@@ -0,0 +1,388 @@
+orcish_rework = {
+	template = "caucasian"
+	
+	skin_color = {
+		10 = { 0.302 0.459 0.329 0.729 } # Green
+	}
+	eye_color = {
+        10 = { 0 0.337 0.259 0.678 } # Yellow, Orange, Red
+		10 = { 0.537 0.329 1 0.686 } # The rest
+	}
+	hair_color = {
+		10 = { 0.18 1 0.18 1 } # Black
+		10 = { 0 0.894 0.059 0.973 } # Brown
```

Comment:
You can bring scripted values from other files. They have more tones.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/107890022-ee793900-6f2f-11eb-85b7-9b17c2a16201.png)

</details>

Reply:
Did this in 715d603825fb5d5d7e0762758538ff3052d8fd63.